src/restruct_poc/system/calculate_Quu.hpp

```diff
@@ -20,19 +26,15 @@ struct CalculateQuu {
         auto x0pupSS = Kokkos::subview(x0pupSS_, i_qp, Kokkos::ALL, Kokkos::ALL);
         auto N_tilde = Kokkos::subview(N_tilde_, i_qp, Kokkos::ALL, Kokkos::ALL);
         auto m1 = Kokkos::Array<double, 9>{};
-        auto M1 = Kokkos::View<double[3][3], Kokkos::MemoryTraits<Kokkos::Unmanaged>>(m1.data());
+        auto M1 = Kokkos::View<double[3][3]>(m1.data());
```

Comment:
Looks like we could use the shorter name `View_3x3` here as defined in `src/restruct_poc/types.hpp`?

Reply:
Yes, it seems like it.  I'll proliferate that change to the other parts of the code affected by this PR.